dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -286,6 +286,15 @@ AppNotificationAssets Microsoft::Windows::AppNotifications::Helpers::GetAssets()
     return assets;
 }
 
+AppNotificationAssets Microsoft::Windows::AppNotifications::Helpers::GetAssets(std::wstring displayName,std::wstring iconFilePath)
+{
+    winrt::check_bool(std::filesystem::exists(iconFilePath));
```

Comment:
What error is thrown if the file doesn't exist?

Reply:
ERROR_FILE_NOT_FOUND